### DISCUSSION TITLE
How would I go about adding another model to the benchmarks?

### DISCUSSION THREAD

**Discussion by ecidon** [microsoft/superbenchmark#570] [Q&A]
If for example I would want to add a Llama model what would I need to do?

**Comment by abuccts** [comment]
Hi @ecidon, you may add the model in [model_benchmarks](https://github.com/microsoft/superbenchmark/tree/main/superbench/benchmarks/model_benchmarks) by adding a class and registering it, or add it in [docker_benchmarks](https://github.com/microsoft/superbenchmark/tree/main/superbench/benchmarks/docker_benchmarks) by providing a docker image and entrypoint script. Here're the example PRs on [model benchmark](https://github.com/microsoft/superbenchmark/pull/582/files) and [docker benchmark](https://github.com/microsoft/superbenchmark/pull/338/files) for your reference.
